resource-managers/kubernetes/docker-minimal-bundle/src/main/docker/spark-base/entrypoint.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Check whether there is a passwd entry for the container UID
+myuid=$(id -u)
+mygid=$(id -g)
+uidentry=$(getent passwd $myuid)
+
+# If there is no passwd entry for the container UID, attempt to create one
+if [ -z "$uidentry" ] ; then
+    if [ -w /etc/passwd ] ; then
+        echo "$myuid:x:$myuid:$mygid:anonymous uid:$SPARK_HOME:/bin/false" >> /etc/passwd
```

Comment:
@erikerlandson why does OpenShift require an entry in `passwd` ?  Is this an additional requirement unique to OpenShift when running docker images?